pagopa/io-app
# fix(Firma con IO): [SFEQS-1342] Update abort signature flow bottom sheet copy

## Short description
This PR updates the FCI copy of the abort signature flow bottom sheet.

## List of changes proposed in this pull request
- Updates the locale for both IT and EN translation;
- Adds a selector for the signature `dossier_title` parameter; 
- Edits the bottom sheet string parsed by the `Markdown` component.

## How to test
- Start the dev server with the following configuration: 
```
...
fci: {
      waitForSignatureCount: 1,
      rejectedCount: 0,
      expiredCount: 0,
      expired90Count: 0,
      waitForQtspCount: 0,
      signedCount: 0,
      noSignatureFieldsCount: 0
    },
...
```
- Run `yarn generate`;
- Start the app by pointing to the dev server and start a signature flow from the message; 
- Press `Go to the documents` and then press `Cancel`;
- The bottom sheet copy should be updated accordingly.
